install.py

```diff
@@ -145,6 +145,14 @@ def go(ui, args, answerfile_address, answerfile_script):
         elif opt == "--netinstall":
             results['netinstall'] = True
             logger.log("This is a netinstall.")
+        # XCP-ng: no-repo-gpgcheck
+        elif opt == "--no-repo-gpgcheck":
+            results['repo-gpgcheck'] = False
+            logger.log("Commandline: no gpg check for yum repository.")
```

Comment:
Well, `Commandline:` prefix is not used in the other cases. It doesn't seem very symmetrical. :slightly_smiling_face:

Reply:
Right, I felt that the other messages were not quite informative enough.  In fact the only upstream log on a boot parameter is `Keymap specified on command-line`, so for consistency that should rather be something like `Yum repository gpg check disabled on command-line`  (and we should improve the messages in our other patches).